src/components/AnnotationViewer.vue

```diff
@@ -422,6 +422,23 @@ export default class AnnotationViewer extends Vue {
     if (!vertices) {
       return;
     }
+
+    // Get the current screenshot dimensions
+    const viewBounds = this.map.bounds();
+    const srcWidth = viewBounds.right - viewBounds.left;
+    const srcHeight = viewBounds.bottom - viewBounds.top;
```

Comment:
Minor optimization: I think this could be stored in the component data to avoid recalculating every time we move the mouse ?